WBooks/src/app/screens/BookDetails/screens/BookDetails/styles.js

```diff
@@ -0,0 +1,92 @@
+import { StyleSheet } from 'react-native';
+
+import {
+  white,
+  darkGray,
+  lightGray,
+  lightBackgroundBlue,
+  red,
+  blue,
+  black
+} from '../../../../../constants/colors';
+
+const styles = StyleSheet.create({
+  bookDetails: {
+    backgroundColor: lightBackgroundBlue,
+    height: '100%'
+  },
+  container: {
+    marginHorizontal: 20,
+    marginTop: 20,
+    marginBottom: 5,
+    paddingHorizontal: 30,
+    paddingVertical: 15,
+    paddingBottom: 22,
+    backgroundColor: white,
+    borderRadius: 5,
+    shadowColor: black,
+    shadowOffset: {
```

Comment:
I think we can move all shadows styles to the `colors` file. So you can have
```
export const shadows = {
  shadowOffset: {
      width: 0,
      height: 1
   },
   shadowOpacity: 0.3,
   shadowRadius: 1
}
```

then import on the styles file like
```
container: {
  .
  .
  .
  backgroundColor: white,
  borderRadius: 5,
  ...shadows
}
```